src/main/kotlin/org/wycliffeassociates/otter/common/domain/ImportResourceContainer.kt

```diff
@@ -216,7 +216,7 @@ private fun Project.mapToCollection(type: String, metadata: ResourceMetadata): C
     )
 }
 
-private fun DublinCore.mapToMetadata(dir: File, lang: Language): ResourceMetadata {
```

Comment:
We need to be careful about top level functions. In java, everything is tied to a class and as a result it's obvious where to find methods and such. This was originally meant as a convenience method to be used inside the ImportResourceContainer, and nowhere else. Private is explicit now, and hints at there possibly being a reason for it. For one, it now allows this method to be accessible to all DublinCore objects in the package; and I maybe that's not desired. If it's not desired, it's not obvious as to where the function is found.

I agree that it should be public, and I've been using it elsewhere in my branch. Unfortunately I haven't merged that branch, but essentially, I created a mapper subpackage in domain, and put this in a DublinCoreMapper.kt source file, in which the function IS public there.

https://github.com/WycliffeAssociates/8woc2018-common/blob/jbs-fast-rc-import/src/main/kotlin/org/wycliffeassociates/otter/common/domain/mapper/DublinCoreMapper.kt